courses-app/src/components/Header/Header.jsx

```diff
@@ -1,17 +1,36 @@
-import React from 'react';
+import { React } from 'react';
+import { useNavigate } from 'react-router-dom';
 import Logo from './components/Logo/Logo';
 import Button from '../common/Button/Button';
 import '../../App.css';
 
 function Header() {
-  // const name = 'button';
+  const navigate = useNavigate();
+
   return (
     <table className="header">
       <tr>
         <td width="10%"><Logo /></td>
         <td width="70%"> </td>
-        <td width="10%">Paul</td>
-        <td width="10%"><Button text="Logout" action={() => { console.log('Button click!'); }} /></td>
+        {
+          localStorage.getItem('token') && <td width="10%">{localStorage.getItem('name')}</td>
```

Comment:
ajde malo da koristimo abstrakciju u kodu. zamisli da radis u teamu od vise developera, ovaj code prosto nece imati smisla nekom ko ga vidi a nema iskustva.
Kada radimo neke provere ili uzimamo neke vrednosti najbolje je napisati custom metodu za to. Takve metode obicno nazivamo helper metode i stavljamo ih u neki helper.js fajl. U ovom slucaju koliko vidim proveravamo da li smo ulogovani i nema bas smisla da stalno pisemo localStorage.getitem(token). sta ako zelimo da promenimo to u nesto drugo ili se sada item zove user umesto token. Morali bi da radimo search kroz ceo projekat i menjamo svuda. Iz tog razloga pravimo helper metode kao npr. isAuthentificated()  i onda je pozivamo svuda gde nam treba, a ako zelimo da je izmenimo jednostavno mozemo promeniti kod unutra metode